integration-tests/main/src/test/java/io/quarkus/it/main/OpenTracingTestCase.java

```diff
@@ -20,4 +22,10 @@ public void invokeResource() {
                 .body(is("TEST"));
     }
 
+    @Test
+    public void testVersion() {
+        RestAssured.when().get("/opentracing/version").then()
+                .body(is(not(emptyString())));
```

Comment:
Is this executed also in the native mode? 

Wouldn't it be better to assert on the exact string or use https://github.com/jaegertracing/jaeger-client-java/blob/master/jaeger-client/src/main/java/io/jaegertracing/client/Version.java#L29?

Reply:
Sure yeah, I do that